packages/api/src/db/tempUser.ts

```diff
@@ -34,10 +34,14 @@ export interface TempUser {
 
 export type OptionalTempUser = TempUser | null
 
+export type UserId = string
+export type UserIdWithTempPrefix = string
+export type GenericUserId = UserId | UserIdWithTempPrefix
```

Comment:
seems a bit useless since all of them are string.

Reply:
Was requested by Tomasz.